master/buildbot/db/logs.py

```diff
@@ -155,7 +197,7 @@ def _splitBigChunk(self, content, logid):
         truncline = content[:self.MAX_CHUNK_SIZE]
         while truncline:
             try:
-                truncline.decode('utf-8')
+                truncline.decode(self.master.config.logEncoding)
```

Comment:
Ok I restore back the hard-coded 'utf-8', removing logEncoding param will be done in another patch.